corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -1,163 +1,104 @@
-{% extends "settings/bootstrap2/base_template.html" %}
+{% extends "settings/bootstrap3/base_template.html" %}
 {% load i18n %}
 {% load hq_shared_tags %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static 'jquery-ui-1.8.23-legacy/jquery-ui.min.js' %}"></script>
-    <script src="{% static 'locations/ko/location_tree.async.js' %}"></script>
-    <link rel="stylesheet" href="{% static 'locations/style/manage.css' %}">
+    <script src="{% new_static 'locations/ko/location_tree.async.js' %}"></script>
+{% endblock %}
+
+{% block stylesheets %}{{ block.super }}
+    <link rel="stylesheet" href="{% new_static 'locations/style/manage.css' %}">
 {% endblock %}
 
 {% block js-inline %}{{ block.super }}
-<script>
-
-var LOAD_LOCS_URL = '{{ api_root }}',
-    NEW_LOC_URL = '{% url "create_location" domain %}';
-
-var locs = {{ locations|JSON }};
-var can_edit_root = {{ can_edit_root|yesno:"true,false" }};
-var selected_loc = {{ selected_id|JSON }};
-var hierarchy = {{ hierarchy|JSON }};
-var show_inactive = {{ show_inactive|yesno:"true,false" }};
-
-function loc_edit_url(loc_id) {
-    var template = '{% url "edit_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_archive_url(loc_id) {
-    var template = '{% url "archive_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_unarchive_url(loc_id) {
-    var template = '{% url "unarchive_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
-
-function loc_delete_url(loc_id) {
-    var template = '{% url "delete_location" domain "-locid-" %}';
-    return template.replace('-locid-', loc_id);
-}
```

Comment:
There is a lot of code that's being incorrectly deleted here.  If you view the diff with whitespace ignored, it's more clear: https://github.com/dimagi/commcare-hq/pull/9862/files?w=1

It looks like all the code about deleting locations will be removed by this PR, my guess is there was a bad merge.